components/CheckDocumentation/schema/IndexUnused.tsx

```diff
@@ -69,6 +68,11 @@ const IndexUnusedGuidance: React.FunctionComponent<CheckGuidanceProps> = ({
           </>
         )}
       </p>
+      <p>
+        Note: when deleting expresssion indexes, you may want to run
+        {" "}<code>CREATE STATISTICS</code> to have better query plans that are
+        aware of the expression results.
+      </p>
```

Comment:
I think it would be nice to give more concrete advice here. "You may want to" is technically correct, but leaves the user wondering whether they do or do not want to. Given this is a weird edge case and not commonly understood, I don't think we need to say anything super-specific, but we could share what we know.

So something like,

```tsx
<p>
  Note: Unused expression indexes can still influence query planner statistics and
  affect query planning. When dropping the index, monitor performance and use
  <PGDocsLink path="sql-createstatistics.html">CREATE STATISTICS</PGDocsLink>
  with the expression from this index if necessary.
</p>
```

I guess we could even include `CREATE STATISTICS` DDL (at least in-app) if we really wanted to, but that seems like overkill.

Thoughts?

Reply:
I've updated it to this:
>Note: expression indexes can influence the query planner even if the index is unused. When dropping an expression index, if you notice some queries are slower you may want to [CREATE STATISTICS](https://www.postgresql.org/docs/current/sql-createstatistics.html) for the expression the index previously covered.